iospod: command not found

Run cd ./packages/smooth_app/ios && iospod update Sentry
/Users/runner/work/_temp/dfe67940-ecd6-4448-b536-e6ffd3c74145.sh: line 1: iospod: command not found
Error: Process completed with exit code 127.

--- a/.github/workflows/ios-release.yml
+++ b/.github/workflows/ios-release.yml
@@ -43,7 +43,7 @@ jobs:
         run: ci/pub_upgrade.sh
 
       - name: "Update Sentry, see: https://github.com/openfoodfacts/smooth-app/issues/1300"
-        run: cd ./packages/smooth_app/ios && iospod update Sentry
+        run: cd ./packages/smooth_app/ios && pod update Sentry
       
       - name: Bundle install
         run: cd ./packages/smooth_app/ios && gem install bundler:1.17.3 && bundle install

